matching-core/src/main/java/com/graphhopper/matching/MatchEdge.java

```diff
@@ -20,15 +20,33 @@
 import com.graphhopper.util.EdgeIteratorState;
 
 /**
- *
+ * A MatchEdge is an edge on a MatchSequence - i.e. one of the edges making up the final
+ * map-matched route. This includes the time at which the edge was travelled.
+ * 
  * @author kodonnell
  */
 public class MatchEdge {
```

Comment:
I think I was trying to be consistent with (the original) `MatchResult` as opposed to `MatchedResult` - happy to change all if you wish?